compiler/hash-ast/src/visualise.rs

```diff
@@ -177,7 +177,7 @@ impl NodeDisplay for Type<'_> {
             }
             Type::Ref(ref_ty) => child_branch(&ref_ty.node_display()),
             Type::RawRef(ref_ty) => child_branch(&ref_ty.node_display()),
-            Type::TypeVar(var) => child_branch(&[format!("var \"{}\"", var.name)]),
+            Type::TypeVar(var) => child_branch(&[format!("var {}", var.name)]),
```

Comment:
The visualiser could be hooked up to use `hash_utils::tree_writing`.

Reply:
work begun on `ast-treewriter`